docs/manual.rst

```diff
@@ -321,7 +321,7 @@ Coordinates may also be sliced. `New in version 1.2.14`.
   [(0.0, 0.0)]
 
 The `Point` constructor also accepts another `Point` instance, thereby making
-a copy.
+a shallow copy.
```

Comment:
I don't think shallow copy quite captures the new behavior. How about something like this:

When given a `Point` instance, the `Point` constructor returns a reference to its argument. It does not make a copy.